src/OctoshiftCLI.Tests/bbs2gh/Handlers/MigrateRepoCommandHandlerTests.cs

```diff
@@ -891,7 +893,39 @@ public async Task It_Throws_When_Aws_Bucket_Name_Not_Provided_But_Aws_Secret_Key
             }))
                 .Should()
                 .ThrowAsync<OctoshiftCliException>()
-                .WithMessage("*--aws-access-key*--aws-secret-key*");
+                .WithMessage("*Amazon S3*--aws-bucket-name*");
+        }
+
+        [Fact]
+        public async Task It_Throws_When_Aws_Bucket_Name_Not_Provided_But_Aws_Session_Token_Provided()
```

Comment:
Should we also add a test for the case where a session token is provided but a region is not provided?

Reply:
Not now, because the region is actually optional in our code so it's totally valid to provide the session token but not the region. that being said I can add the test and make sure that we're printing the log message and it future once we made the region required we can just update the test to check for the validation.